v2/mongodb-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/mongodb/options/MongoDbToBigQueryOptions.java

```diff
@@ -94,4 +95,25 @@ public interface BigQueryWriteOptions extends PipelineOptions, DataflowPipelineO
 
     void setOutputTableSpec(String outputTableSpec);
   }
+
+  /** UDF options. */
+  public interface JavascriptDocumentTransformerOptions extends PipelineOptions {
+    @TemplateParameter.Text(
```

Comment:
@TemplateParameter.GcsReadFile

Reply:
+1 to using proper metadata annotations. This feels like extra work for now, but we want to have those annotations as a source of truth for all other resources/documentations.

Reply:
Accepted. Will change. Thanks